editor/templates/place/edit.html

```diff
@@ -280,6 +280,16 @@ <h3>Editing {{ wof_doc.properties['wof:name'] or wof_doc.id }}</h3><small>(<a hr
         <input type="text" class="form-control wof-input" name="edtf:superseded" value="{{ wof_doc.properties['edtf:superseded'] }}">
         <small id="edtf-superseded-help" class="form-text text-muted">Indicates the date when a record was superseded by another record. Supports EDTF date notation like: 2000, 2000-01, 2000-01-01 and approximate dates like ~2000. Defaults to uuuu when unknown.</small>
     </div>
+    <div class="form-group">
+        <label for="wof-superseded_by">Superseded by</label>&nbsp;<small>(<code>wof:superseded_by</code>)</small>
+        <input type="text" class="form-control wof-input" name="wof:superseded_by" value="{{ wof_doc.properties['wof:superseded_by'] }}">
+        <small id="wof-superseded_by" class="form-text text-muted">Indicates the WOF ID of the record that supersedes this record.</small>
+    </div>
+    <div class="form-group">
+        <label for="wof-supersedes">Supersedes</label>&nbsp;<small>(<code>wof:supersedes</code>)</small>
+        <input type="text" class="form-control wof-input" name="wof:supersedes" value="{{ wof_doc.properties['wof:supersedes'] }}">
```

Comment:
According to [the properties repo](https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/wof/supersedes.json), it looks like this should be an array of integers. Can you add a `data-wof-validation-regex="^\d+$"` and `data-wof-type="list"` to validate that in JavaScript:

```suggestion
        <input type="text" class="form-control wof-input" name="wof:supersedes" data-wof-type="list" data-wof-validation-regex="^\d+$" value="{{ wof_doc.properties['wof:supersedes'] }}">
```